apache/superset
# WIP: fix(time_picker): Fix incorrect time range display

### SUMMARY

This fixes #16462. It will correct time range display from `(begin, end)` to `[begin, end)`

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

BEFORE(copied from issue):
![image](https://user-images.githubusercontent.com/5211253/146664969-733c9bd6-5d84-4f98-bb10-611966474a3a.png)

AFTER:
![image](https://user-images.githubusercontent.com/5211253/146665273-07714877-bd96-4326-a866-1cc38ab35c30.png)

### TESTING INSTRUCTIONS

Just create a dashboard with a time range filter and confirm time range is `[begin, end)`

### ADDITIONAL INFORMATION


- [X] Has associated issue: Fixes #16462
